app/models/info_request.rb

```diff
@@ -1430,6 +1430,8 @@ def accept_incoming?(email, raw_email_data)
     will_be_rejected = (response_rejector && response_rejection) ? true : false
 
     if will_be_rejected && response_rejection.reject(response_rejector.reason)
+      self.rejected_incoming_count += 1
```

Comment:
Could use [`#increment!`](http://api.rubyonrails.org/v3.2.22/classes/ActiveRecord/Persistence.html#method-i-increment-21) here – I don't think running the validations is important after this?